src/solidity-frontend/solidity_convert.cpp

```diff
@@ -3062,6 +3233,17 @@ bool solidity_convertert::get_type_description(
 
     break;
   }
+  case SolidityGrammar::TypeNameT::MappingTypeName:
+  {
+    // e.g.
+    //  "typeIdentifier": "t_mapping$_t_uint256_$_t_string_storage_$",
+    //  "typeString": "mapping(uint256 => string)"
+    // since the key will always be regarded as string, we only need to obtain the value type.
+
+    typet val_t;
+    //!TODO
```

Comment:
Can you explain here for this TODO? What should be done?

Reply:
That is the unfinished mapping codes. Will add a log_error and abort